prow/plank/reconciler.go

```diff
@@ -52,6 +52,11 @@ import (
 
 const ControllerName = "plank"
 
+// PodStatus constants
```

Comment:
The additions here are copies of code that was in controller.go but also used by the code in reconciler.go